package.json

```diff
@@ -152,6 +153,7 @@
     "react-native-launch-arguments": "^4.0.2",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
+    "react-native-nitro-modules": "^0.17.0",
```

Comment:
Why do we need to import this lib to the main App repo?

Reply:
Yes, we use it here `https://github.com/Expensify/App/pull/51018/files#diff-f1bb68e32ce747ed73a8d72d8ebcdd6e657539f61e7cb330d5a340a6f5ee0e42R66`